src/spyglass/micro_ct/v1/micro_ct.py

```diff
@@ -0,0 +1,120 @@
+    -> [nullable] LabMember.proj(scanner_operator='user_name') # Optional: Who operated the scanner?
+    """
+
+
+@schema
+class MicroCTImages(SpyglassMixin, dj.Computed):
+    definition = """
+    # Links MicroCTScan info to the Analysis NWB file containing the image volume
+    -> MicroCTScan
+    images_id: varchar(32)              # User-defined ID for these images (e.g., scan_id)
+    ---
+    -> AnalysisNwbfile                  # Link to the NWB file storing image data
+    processing_time=CURRENT_TIMESTAMP: timestamp # Timestamp of NWB file creation
+    processing_notes="": varchar(1024)  # Notes on image processing applied before/during NWB creation
+    """
+
+    # Ensure this key is unique for MicroCTImages entries
+    # key_source = MicroCTScan
+
+    def make(self, key: dict) -> None:
+        """
+        Populate MicroCTImages table.
+        This method should:
+        1. Find the raw reconstructed image data using `raw_scan_path` from `MicroCTScan`.
+        2. Process the images if necessary (e.g., format conversion, cropping).
+        3. Create an NWB file containing the image volume (e.g., using `pynwb.image.ImageSeries`).
+           - Store relevant metadata (voxel sizes from MicroCTScan, etc.) within the NWB file.
+        4. Create an `AnalysisNwbfile` entry for the new NWB file.
+        5. Insert the key, including the `analysis_file_name` from AnalysisNwbfile,
+           and any processing notes into this table.
+        """
+        logger.info(f"Populating MicroCTImages for key: {key}")
+        # Placeholder: Replace with actual NWB creation and insertion logic
+        # Example steps (conceptual):
+        # 1. scan_entry = (MicroCTScan & key).fetch1()
+        # 2. raw_path = scan_entry['raw_scan_path']
+        # 3. image_data, metadata = process_microct_images(raw_path, scan_entry) # Your function
+        # 4. nwb_file_name = f"{key['subject_id']}_{key['scan_id']}_images.nwb"
+        # 5. nwb_file_path = AnalysisNwbfile().create(nwb_file_name)
+        # 6. create_microct_nwb(nwb_file_path, image_data, metadata) # Your function
+        # 7. AnalysisNwbfile().add(key['subject_id'], nwb_file_name)
+        # 8. self.insert1({
+        #     **key,
+        #     'images_id': key['scan_id'], # Or generate a new unique ID if needed
+        #     'analysis_file_name': nwb_file_name,
+        #     'processing_notes': '...',
+        # })
+        pass
+
+
+@schema
+class MicroCTRegistration(SpyglassMixin, dj.Manual):
+    definition = """
+    # Stores results/params of aligning microCT image data to a target coordinate system
+    -> MicroCTImages                    # Link to the source microCT NWB file info
+    registration_id: varchar(32)        # Unique ID for this registration instance/parameters
+    ---
+    -> CoordinateSystem                 # The TARGET coordinate system (e.g., 'allen_ccf_v3_ras_um')
+
+    # --- Registration Parameters ---
```

Comment:
As above, maybe protocol/param tables - it might be the case that a generalized registration table could be a params table for both schemas